mix.exs

```diff
@@ -21,8 +21,7 @@ defmodule HPack.Mixfile do
 
   defp deps do
     [
-      {:earmark, "~> 0.1", only: :dev},
-      {:ex_doc, "~> 0.11", only: :dev},
+      {:ex_doc, ">= 0.0.0", only: :dev},
```

Comment:
Shouldn't donwgrade anything I think, just loosen the dependency. The one in `mix.lock` will be installed. I can change it back if you prefer though.